### PR TITLE
Use AVAILABLE_LOCALES from new renault_api package

### PR DESCRIPTION
**Pull request by epenet**

### CODE REVIEW

**Comment by epenet** [comment]
Fix #70

File: custom_components/renault/strings.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
       "kamereon_no_account": "Unable to find Kamereon account."
```
Comment (oncleben31) [inline custom_components/renault/strings.json:4]: From a general point of view, this file is useless in a custom component. Do we need to keep it here even if we plan to merge in Home Assistant core?

File: custom_components/renault/translations/en.json [hunk 2]
```diff
@@ -4,8 +4,7 @@
       "kamereon_no_account": "Unable to find Kamereon account."
     },
     "error": {
-      "invalid_credentials": "Invalid credentials.",
-      "preload_failed": "Unable to preload Renault keys."
+      "invalid_credentials": "Invalid credentials."
```
Comment (oncleben31) [inline custom_components/renault/translations/en.json:7]: I'm not sure but perhaps we should use the common strings here To reuse the system translation already done. No ?
```
"invalid_credentials": "[%key:common::config_flow::error::invalid_auth%]"
```

File: custom_components/renault/translations/fr.json [hunk 3]
```diff
@@ -4,7 +4,6 @@
       "kamereon_no_account": "Aucun compte Kamereon trouvé"
     },
     "error": {
-      "invalid_api_keys": "Clé API invalide",
       "invalid_credentials": "Authentification invalide"
```
Comment (oncleben31) [inline custom_components/renault/translations/fr.json:7]: cf. comment for `en.json`

File: requirements.txt [hunk 4]
```diff
@@ -1 +1,2 @@
 PyZE==0.6.0
+renaut-api==0.0.1a0
```
Comment (oncleben31) [inline requirements.txt:2]: Why to you maintain a `requirement.txt` file ? It is not required in a custom component and I don't see any CI or tests that could make usage of this.

**Review by oncleben31 — COMMENTED** [review]
Lot of open questions ;-)

File: custom_components/renault/strings.json (hunk 1 above)
Comment (epenet) [reply custom_components/renault/strings.json:4]: I think it is better to keep it for the future migration.

File: custom_components/renault/translations/en.json (hunk 2 above)
Comment (epenet) [reply custom_components/renault/translations/en.json:7]: I think the common strings are generated by HA core during the "publishing" time, so they cannot be used inside custom components.

File: custom_components/renault/translations/fr.json (hunk 3 above)
Comment (epenet) [reply custom_components/renault/translations/fr.json:7]: I think the common strings are generated by HA core during the "publishing" time, so they cannot be used inside custom components.

File: requirements.txt (hunk 4 above)
Comment (epenet) [reply requirements.txt]: Removed

File: custom_components/renault/strings.json (hunk 1 above)
Comment (oncleben31) [reply custom_components/renault/strings.json:4]: OK

**Review by oncleben31 — APPROVED** [review]
LGTM

**Pull request merged**